lib/utilities.js

```diff
@@ -13,14 +13,12 @@ function invertObject(obj, override) {
     throw new TypeError('First argument must be an object');
   }
 
-  var key;
-  var value;
   var isOverridePresent = typeof override === 'function';
   var overrides = {};
   var result = {};
 
-  for (key in obj) {
-    value = obj[key];
+  for (var key in obj) {
+    var value = obj[key];
```

Comment:
I think it would be clearer to declare it in the for statement block. 🙏

Reply:
There is a premature optimization when the variable is initialized once, but it is a minor so I am fine with your change